requirements/template.txt

```diff
@@ -3,9 +3,13 @@
 
 # add what breaks on RTD
 
-wrapanapi~=3.0.21
+wrapanapi~=3.1.0
 
 --no-binary pycurl
 pycurl
 
+ansible
```

Comment:
The freeze script was trying to remove these because they weren't in the template.